sdk/resourcemanager/azure-resourcemanager-cosmos/src/test/java/com/azure/resourcemanager/cosmos/CosmosDBTests.java

```diff
@@ -64,6 +64,7 @@ protected void cleanUpResources() {
     }
 
     @Test
+    @Disabled("Local authentication methods are not allowed.")
```

Comment:
Find some way to enable the test then.

Reply:
I see it reverted.

What is the cause of prior problem?

Reply:
Got it.

But we should not silently change behavior in SDK. See https://github.com/Azure/azure-sdk-for-java/pull/45273#discussion_r2086346036